[IMP]Misc:SysParm-edits-email-doc (#4277)
odoo/documentation: content/applications/general/email_communication/email_servers.rst

parameters were introduced in Odoo 16 to prevent aliases from creating too many records and to
prevent feedback loops on the catchall reply-to email address. They are present in database but not
in the *System Parameters*. To override the following defaults they need to be added in.

The two system parameters are as follows:

- `mail.incoming.limit.period` (60 minutes by default)
- `mail.incoming.limit.alias` (5 by default)

Add these fields in Odoo by navigating to (in :ref:`developer mode <developer-mode>`)
:menuselection:`Settings --> Technical Menu --> Parameters --> System Parameters`. Change the value
of the parameter so that it fits the company's needs.

When an email is received in the Odoo database on the catchall email address or on any alias, Odoo
looks at the mail received for the given period of time defined in the system parameter
`mail.incoming.limit.period`. If the received email was sent to an alias then Odoo will reference
the `mail.incoming.limit.alias` system parameter and determine the value as the number of records
this alias is allowed to create in the give period of time (value of `mail.incoming.limit.period`).

> In addition when email is received to the catchall email address Odoo will reference the emails

Reviewer: comma
```suggestion
In addition, when email is received to the catchall email address, Odoo will reference the emails
```

received to the database during the set period of time (as stated by the value in the system
parameter: `mail.incoming.limit.period`). Odoo will then determine whether any of the emails

> received match that of the email being received during that time-frame and prevent a feedback loop
> from occurring if a duplicate email should be present.

Reviewer: suggestion around wording
```suggestion
received match that of the email(s) being received during the specified time-frame, and will prevent
a feedback loop from occurring if a duplicate email is detected.
```
